File: projects/m1/014-cell-phone-bill/solutions/stefandifede/cell-phone-bill.js
const minutesUsed = prompt("Insert the amount of minutes used")
const textMessagesUsed = prompt("Insert the amount of text message sent")
const callsTo911 = prompt("Insert number of calls to 911 call center")
const basicBill = 15


let callsTo911Charge = 0.44 * callsTo911;
let extraMinutesCharge = 0;
let extraMessages = 0;

function calculateExtraMinutes() {
    if (minutesUsed > 50) {
        extraMinutesCharge = (0.25 * (minutesUsed - 50));
    } else {extraMinutesCharge = 0}
}

function calculateExtraMessages () {
    if (textMessagesUsed > 50) {
        extraMessages = (0.15 * (textMessagesUsed - 50));
    } else {extraMessages = 0}
}

calculateExtraMessages();
calculateExtraMinutes();


const totalExtraCharge = (extraMinutesCharge + extraMessages + callsTo911)
const totalExtraChargeTax = ((5 * totalExtraCharge) / 100); 

const totalBill = basicBill + extraMinutesCharge + extraMessages + callsTo911Charge + totalExtraChargeTax;

const monthBill = alert("Basic bill: 15$" + "\n Additional minutes charge: " + extraMinutesCharge.toFixed(2) + " $" + "\n Additional messages charge: " + extraMessages.toFixed(2) + " $" + "\n 911 calls charge: " + callsTo911Charge.toFixed(2) + " $" + "\n Total Tax: " + totalExtraChargeTax.toFixed(2) + " $" + "\n Total bill: " + totalBill.toFixed(2) + " $")

File: projects/m1/014-cell-phone-bill/solutions/GabrieleGabellone/cell_phone_bill.py
#cell-phone-bill

minutes_used = int(input("Enter the number of minutes used in the month: "))
txt_used = int(input("Enter the number of text messages used in the month: "))
add_minutes_charge = add_txt_charge = 0
if minutes_used > 50:
    add_minutes_charge = (minutes_used - 50) * 0.25
if txt_used > 50:
    add_txt_charge = (txt_used - 50) * 0.15
sales_tax = (15 + add_minutes_charge + add_txt_charge + 0.44) * (5/100)
total_bill_amount = 15 + add_minutes_charge + add_txt_charge + 0.44 + sales_tax
print("-The base charge is €15.00")
if add_minutes_charge > 0: print("-The additional minutes charge is €{:.2f}".format(add_minutes_charge))
if add_txt_charge > 0: print("-The additional text message charge is €{:.2f}".format(add_txt_charge))
print("-The 911 fee is €0.44")
print("-The sales tax is €{:.2f}".format(sales_tax))
print("Total bill amount: €{:.2f}".format(total_bill_amount))